assembly/std/context.ts

```diff
@@ -61,7 +61,7 @@ export function caller(): Address {
  *
  * Callee is the smart contract of the pending function.
```

Comment:
```suggestion
 * Callee is the current smart-contract address.
```

Maybe we should create an alias named `currentAddress`